pkgs/development/tools/analysis/rizin/cutter.nix

```
@@ -24,15 +32,42 @@ let cutter = mkDerivation rec {
     fetchSubmodules = true;
   };
 
-  nativeBuildInputs = [ cmake qmake pkg-config python3 wrapQtAppsHook ];
-  propagatedBuildInputs = [ python3.pkgs.pyside2 ];
-  buildInputs = [ graphviz qtbase qttools qtsvg qtwebengine rizin python3 ];
+  patches = [
+    # tracking: https://github.com/rizinorg/cutter/pull/3268
+    (fetchpatch {
+      url = "https://github.com/rizinorg/cutter/commit/3d12ca3aa613d7adc095182a2e556d7de9471145.patch";
+      hash = "sha256-nisvbOODVPSLjiLOjYH8N7wYeouOB6cZCjF15n3p7UI=";
+    })
```

```suggestion
    (fetchpatch {
      name = "cutter-simplify-python-binding-include-handling.patch";
      url = "https://github.com/rizinorg/cutter/compare/7256fbb00e92ab12a24d14a92364db482ed295cb..ca5949d9d7c907185cf3d062d9fa71c34c5960d4.diff";
      hash = "sha256-bqV2FTA8lMNpHBDXdenNx+1cLYa7MH47XKo1YatmLV4=";
    })
```

Use all three commits from upstream PR now